lib/hotel_manager.rb

```diff
@@ -0,0 +1,85 @@
+require 'date'
+require 'pry'
+
+module Hotel
+  class HotelManager
+
+    attr_reader :room_list
+    attr_accessor :reservations
+
+    def initialize
+      @room_list = load_rooms
+      @reservations = Array.new
+      # puts "A new hotel manager has been created."
+    end
+
+    def load_rooms
+      room_list = Array(1..20)
+      return room_list
+    end
+
+    def get_reservation_id
+
+      if @reservations == []
+        next_reservation_id = 1
+        # puts "The next reservation will be #{next_reservation_id}"
+        return next_reservation_id
+      else
```

Comment:
This else logic will actually still work if the array is empty so you don't need the conditional